static/noscript.ts

```diff
@@ -22,47 +22,39 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-require('./noscript.scss');
+import $ from 'jquery';
```

Comment:
You taught me that this was globally defined :)

Reply:
Yep, but I think this might be necessary because noscript.ts is its own entry in the webpack compilation, which means that it wouldn't have jQuery imported anywhere else